### DISCUSSION TITLE
Will Transportation receive any love again?

### DISCUSSION THREAD

**Discussion by Psy-Virus** [MrTJP/ProjectRed#1808] [Q&A] [answered]
Hey there,
I've not been active here since November 2018 cause I was patiently waiting for the promised transportation rewrite XD

Have I been patient enough to ask for the transportation rewrite status? 😁

Will it ever receive love again?

Thanks in advance :)

**Comment by Olivercola22** [comment]
I would like to know too.

I was fascinated with this mod when i was younger. I love to see items flow through the pipes. And the idea of having a storage system which would fill about the same space as normal one but with the added bonus of being able to request and look for items in one spot.

Hopefully they restart the mod so we can again be happy.

**Comment by MrTJP** [comment, marked as answer]
Would love to revisit the routing pipes again. That's actually the next major thing planned once I get through the few remaining planned features for Fabrication.
